src/helpers.js

```diff
@@ -1,3 +1,17 @@
+function isNulledBox(box) {
+  return !box.w && !box.h && !box.x && !box.y
+}
+
+function domContains(node) {
```

Comment:
But talking about helpers, we could provide a plugin with helpers that are specifically target SVG usage. I would personally have a lot of fun, writing Applicatives and Functors for SVG.